.github/workflows/build.yaml

```diff
@@ -43,6 +43,34 @@ jobs:
         with:
           error-on: '"note"'
 
+  html5-check:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'devel'
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: rcmdcheck
+
+      - name: Install pdflatex
+        run: sudo apt-get install texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra
+
+      - name: Install tidy
+        run: sudo apt install tidy
+
+      - uses: r-lib/actions/check-r-package@v2
+        with:
+          args: '"--as-cran"'
```

Comment:
default value is `--as-cran --no-manual` so I'm just re-enabling the manual check here